src/_includes/feature_lists/cloud.njk

```diff
@@ -2,47 +2,75 @@
     set sections = [{
         label: "Features",
         rows: [{
+            id: 'hosting',
             label: "Hosting",
-            values: ['Self Managed', 'FlowForge- or Self Managed']
+            values: ['Self Managed', 'FlowForge- or Self Managed'],
+            info: "<p>FlowForge can be deployed using Docker, Kubernetes or run the core FlowForge platform on your local machine directly.</p><p>FlowForge Cloud provides you with a browser-based, secure and cloud-based platform, with all of the FlowForge Premium features included and managed by us to reduce your operational overheads.</p>"
         }, {
+            id: 'multiple-projects',
             label: "Multiple Projects",
-            values: ['check', 'check']
+            values: ['check', 'check'],
+            info: "<p>Scale your Node-RED infrastructure with as many Node-RED instances as you need. In FlowForge, a Project manages your instance of Node-RED, attached devices, context and environment variables and version history.</p>"
         }, {
+            id: 'node-red-upgrades',
             label: "Easy Node-RED Upgrades",
-            values: ['check', 'check']
+            values: ['check', 'check'],
+            info: "<p>FlowForge Stacks enable single-click upgrading of Node-RED as new versions are released.</p><p>Our intuitive UI makes it clear when projects are running out-of-date stacks, so you have piece of mind over your flows running the latest and greatest versions that you've approved."
```

Comment:
I wonder if the mechanism needs to be so prominent, as it might confuse customers?

Reply:
Fair - open to alternative description

Reply:
The benefits:
- Admin gets to approve Node-RED versions, makes upgrade available
- User: Understanding there's a newer version available
- Both: Insights into versioning and the easy path is the right path

Also; UX wise it's not a single click :) And missing a closing `</p>`

```suggestion
            info: "<p>Enable users to upgrade their Node-RED with a few clicks as new versions are released.</p><p>Our intuitive UI makes it clear when projects are running out-of-date stacks, so you have piece of mind over your flows running the latest and greatest versions that you've approved.</p>"
```